test/tc_webhook_config_test.go

```diff
@@ -97,3 +97,18 @@ func (e *e2e) getWebhookAttribute(hook, attr string) string {
 		"spo-mutating-webhook-configuration",
 		"--output", "jsonpath="+jsonPath)
 }
+
+func (e *e2e) testCaseWebhookHTTP([]string) {
+	if !e.testWebhookHTTP {
+		e.T().Skip("Skipping webhook HTTP version related tests")
+	}
+	e.logf("Test webhook HTTP version")
+
+	curNs := e.getCurrentContextNamespace(config.OperatorName)
+	metricsEndpoint := fmt.Strintf("https://localhost:9443/metrics-spod", curNs)
```

Comment:
@Vincent056 @saschagrunert Do you have any tips how to test the mutating webhooks?

Reply:
Hm, may I ask you to fix the CI first? 

```
test/tc_metrics_test.go:135:40: undefined: config
test/tc_metrics_test.go:136:25: undefined: fmt.Strintf
test/tc_webhook_config_test.go:107:40: undefined: config
test/tc_webhook_config_test.go:108:25: undefined: fmt.Strintf (typecheck
```

Reply:
maybe you can test this endpoint `https://webhook-service.<ns>.svc:443/`

Reply:
@Vincent056 Thanks, that worked.

Now I need figure out why the Metrics endpoints are not  HTTP/1.1

Reply:
The `kube-rbac-proxy-image` was leading the Metrics endpoint to be HTTP/2.